src/main/java/org/jboss/qa/jenkins/test/executor/utils/MavenCli.java

```diff
@@ -238,10 +238,9 @@ public boolean isFailAtEnd() {
 	}
 
 	public int run() throws Exception {
-
 		final ProcessBuilder processBuilder = new ProcessBuilder(generateCommand());
-		processBuilder.redirectOutput(ProcessBuilder.Redirect.INHERIT);
-		processBuilder.redirectError(ProcessBuilder.Redirect.INHERIT);
+		//processBuilder.redirectOutput(ProcessBuilder.Redirect.INHERIT) doesn't work on HPUX.
```

Comment:
@mswiech please remove from here, put all info to javadoc of RedirectProcessOut.